### PR TITLE
ONE Vraska, Betrayal's Sting & Jace, the Perfected Mind

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/vraska_betrayals_sting.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Loyalty:6
+K:Compleated
+A:AB$ Draw | Cost$ AddCounter<0/LOYALTY> | Planeswalker$ True | SubAbility$ DBLoseLife | SpellDescription$ You draw a card and you lose 1 life. Proliferate.
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ 1 | SubAbility$ DBProliferate 
+SVar:DBProliferate:DB$ Proliferate 
+A:AB$ Animate | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Creature | TgtPrompt$ Select target creature | RemoveAllAbilities$ True | Colors$ Green | Types$ Artifact,Treasure | Abilities$ TreasureSac |  RemoveCreatureTypes$ True | RemoveCardTypes$ True | Duration$ Permanent | SpellDescription$ Target creature becomes a Treasure artifact with "{T}, Sacrifice this artifact: Add one mana of any color" and loses all other card types and abilities.
+SVar:TreasureSac:AB$ Mana | Cost$ T Sac<1/CARDNAME> | Produced$ Any | SpellDescription$ Add one mana of any color.
+A:AB$ Poison | Cost$ SubCounter<9/LOYALTY> |  ValidTgts$ Player | TgtPrompt$ Select target player | ConditionCheckSVar$ X | ConditionSVarCompare$ LE9 | Num$ Difference | SpellDescription$ If target player has fewer than nine poison counters, they get a number of poison counters equal to the difference.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vraska_betrayals_sting.txt:11]: LT9

File: forge-gui/res/cardsfolder/upcoming/vraska_betrayals_sting.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Name:Vraska, Betrayal's Sting
+ManaCost:4 B PB
+Types:Legendary Planeswalker Vraska
+Loyalty:6
+K:Compleated
+A:AB$ Draw | Cost$ AddCounter<0/LOYALTY> | Planeswalker$ True | SubAbility$ DBLoseLife | SpellDescription$ You draw a card and you lose 1 life. Proliferate.
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ 1 | SubAbility$ DBProliferate 
+SVar:DBProliferate:DB$ Proliferate 
+A:AB$ Animate | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Creature | TgtPrompt$ Select target creature | RemoveAllAbilities$ True | Colors$ Green | Types$ Artifact,Treasure | Abilities$ TreasureSac | RemoveCreatureTypes$ True | RemoveCardTypes$ True | Duration$ Permanent | SpellDescription$ Target creature becomes a Treasure artifact with "{T}, Sacrifice this artifact: Add one mana of any color" and loses all other card types and abilities.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vraska_betrayals_sting.txt:9]: You shouldn't need RemoveCreatureTypes

The engine should handle that rules part already

File: forge-gui/res/cardsfolder/upcoming/jace_the_perfected_mind.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Name:Jace, the Perfected Mind
+ManaCost:2 PU U 
+Types:Legendary Planeswalker Jace
+Loyalty:5
+K:Compleated
+A:AB$ Pump | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | NumAtt$ -3 | IsCurse$ True | Duration$ UntilYourNextTurn | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | SpellDescription$ Until your next turn, up to one target creature gets -3/-0.
+A:AB$ Mill | Cost$ SubCounter<2/LOYALTY> | NumCards$ 3 | ValidTgts$ Player | TgtPrompt$ Select a player | SubAbility$ DBDraw | SpellDescription$ Target player mills three cards. Then if a graveyard has twenty or more cards in it, you draw three cards. Otherwise, you draw a card.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/jace_the_perfected_mind.txt:7]: `TgtPrompt$ Select target player`

File: forge-gui/res/cardsfolder/upcoming/jace_the_perfected_mind.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+SVar:DBDraw:DB$ Draw | NumCards$ Y
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/jace_the_perfected_mind.txt:8]: I think this will need Stack and/or SpellDesc

**Review by tool4ever — APPROVED** [review]
looks good, thanks

**Comment by Hanmac** [comment]
@Simisays 
* Jace, the Perfected Mind
* Urtet, Remains of Memnarch
* Vraska, Betrayal's Sting

check this cards for " " at the end of the lines

**Pull request merged**